src/ctapipe/irf/vis_utils.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
```

Comment:
The commit message says "Fixed some quality issues in sonar" but it also adds a couple of new files. So I think that was probably by mistake.

Reply:
No, part of the things sonar raised was code duplication, so collected various repeated snippets into new functions and put those into a separate files.